taskcluster/docker/base-android/install-sdk.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script is also called at each run of the decision task
+
+set -ex
+
+export ANDROID_SDK_SHA256='444e22ce8ca0f67353bda4b85175ed3731cae3ffa695ca18119cbacef1c1bea0'
+export ANDROID_SDK_VERSION='3859397'
```

Comment:
Centralizing the installation of the android sdk makes bumps easier. This line is now the only place where `3859397` is hardcoded.